_data/library/scalalibs.yml

```diff
@@ -273,3 +273,8 @@
       dep: |
         addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full)
         libraryDependencies += "com.thoughtworks.enableIf" %% "enableif" % "latest.release"
+    - name: sbt-node
+      url: https://github.com/jokade/sbt-node
+      desc: A Sbt plugin for integration with npm / node.js based build tools.
+      dep: 'addSbtPlugin("de.surfice" % "sbt-node" % "latest.release")'
+
```

Comment:
Make sure that there is no blank line here. There should be 1 EOL at the end of the file, but now the file has 2 EOLs.

Also: same thing with the version number.